charts/hypha/values.yaml

```diff
@@ -59,25 +59,22 @@ redis:
   enabled: enabled
   global:
     redis:
-      password: "admin"
+      password: "redis-password"
```

Comment:
Is there security issue here if we set the password here? Maybe move to the values file?